### ISSUE TITLE
MD057 incorrectly flags absolute links to directories

### ISSUE THREAD

**Issue by cespare** [rvben/rumdl#632]
### rumdl version

0.1.94

### Installation method

Pre-built binary

### Bug description

Rumdl can't seem to resolve absolute links when they point to directories (that exist).

Absolute links to files work. Relative links to directories work.

### Steps to reproduce

I have a test directory in /tmp with these files (only):

- `d/` directory
- `d/foo.md` (empty file)
- `rumdl.toml`:
  ```
   enable = ["MD057"]
   [MD057]
   absolute-links = "relative_to_roots"
   ```
- `x.md`:
   ```
   This is an [absolute dir](/d).
   This is a [relative dir](d).
   
   This is an [absolute file](/d/foo.md).
   This is a [relative file](d/foo.md).
   ```

In this directory, I'm running `rumdl check`.

### Expected behavior

No errors/warnings.

### Actual behavior

```
$ rumdl check
x.md:1:27: [MD057] Absolute link '/d' was not found under the project root

Issues: Found 1 issues in 1/2 files (2ms)
```

### Configuration

```toml
n/a
```

### Operating system

Linux

**Comment by rvben** [comment]
Fixed in **v0.1.96**. With `absolute-links = "relative_to_roots"`, MD057 now accepts absolute links that resolve to an existing directory (e.g. `/d`), so they are no longer flagged. Closing as released.

**Comment by cespare** [comment]
Thanks @rvben!
